pom.xml

```diff
@@ -97,43 +98,36 @@
         <groupId>org.apache.maven</groupId>
         <artifactId>maven-artifact</artifactId>
         <version>${mavenVersion}</version>
-        <scope>provided</scope>
```

Comment:
Lower level modules may scope provided. If provided scope in depMgmt causes issues in CLI.